gems/sorbet-runtime/lib/types/private/decl_state.rb

```diff
@@ -16,4 +16,14 @@ def self.current=(other)
   def reset!
     self.active_declaration = nil
   end
+
+  def self.without_on_method_added
+    begin
+      old_value = current.skip_on_method_added
+      current.skip_on_method_added = true
+      yield
+    ensure
+      current.skip_on_method_added = old_value
+    end
+  end
```

Comment:
Sorry I mispoke. I think this should be an instance method, so that it's `current.without`